hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -36,8 +36,6 @@
 @InterfaceAudience.Public
 @InterfaceStability.Unstable
 public final class OzoneConfigKeys {
-  public static final String OZONE_TAGS_SYSTEM_KEY =
-      "ozone.tags.system";
   public static final String DFS_CONTAINER_IPC_PORT =
```

Comment:
For my understanding, Is there a reason why this config needed to be taken out of OzoneConfigKeys and added in OzoneConfiguration?

Reply:
Thanks @SaketaChalamchala for the review.

`TestOzoneConfigurationFields` verifies that all keys defined in `OzoneConfigKeys` (and some others) are also present in `ozone-default.xml` (and vice versa).  It would have been enough to add as an exception in `TestOzoneConfigurationFields`.